rules/integrations/aws/privilege_escalation_sts_temp_creds_via_assume_role.toml

```diff
@@ -1,90 +0,0 @@
-[metadata]
-creation_date = "2021/05/17"
-integration = ["aws"]
```

Comment:
Shouldnt this be deprecated properly? 

Like Rename this release and move to deprecated folder next release

Reply:
File renames